<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QTableView" name="tableViewVersions"/>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButtonCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/cross.png</normaloff>../images/cross.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButtonOpenImage">
     <property name="text">
      <string>Open Image</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../icons/database_edit.png</normaloff>../icons/database_edit.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBoxMostRecent">
     <property name="text">
      <string>Always open most recent version</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
